src/docs/concepts-development/headless-content-management/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Headless content management
+slug: headless-content-management
+updated: 2023-01-11
+category: concepts-development
+ingress:
+  This article explains how data can be edited through the Page Editor
```

Comment:
@SariSaar one piece of feedback we received from Juho is that Page Editor and Page Manager are internal terms. We should just use "Pages" to describe the Console UI

Reply:
@thomas-rocca Acknowledged, I'll rephrase this!